dilshod/xlsx2csv
# sheet delimiter, line terminator and output encoding supported

add options:
 - sheet delimiter (by https://github.com/rulnick)
 - line terminator
 - output encoding (Python3 only)

revise and add test:
 - test script can be used on Windows now
 - verify various delimiters
 - verify utf8 output 
 - all tests passed

other misc update:
 - update readme corresponding to changes above
 - variable name `format` is changed to `format_str` (former name collided with buildin func)
